eoscostarica/lifebank
# Feat/tokens transaction history

Resolves #705 

### What does this PR do?
Create the option to a lifebank to see the tokens transaction

### How should this be tested?
- Make run
- Make a lifebank transaction
- Redeem an offer
- Check tokens history in lifebank profile
